driver-scala/src/test/scala/org/mongodb/scala/AggregateObservableSpec.scala

```diff
@@ -49,17 +49,6 @@ class AggregateObservableSpec extends BaseSpec with MockFactory {
     val ct = classOf[Document]
     val verbosity = ExplainVerbosity.QUERY_PLANNER
 
-    wrapper.expects(Symbol("allowDiskUse"))(true).once()
```

Comment:
the style of the test changed a bit, but was done in a semi automated way. I managed to use a regex to transform
```scala
 wrapper.expects(Symbol("allowDiskUse"))(true).once()
```
into
```scala
verify(wrapper).allowDiskUse(true)
```
and then move the lines to "after" the test instead of before